tree/ntupleutil/v7/src/RNTupleInspector.cxx

```diff
@@ -166,8 +166,8 @@ ROOT::Experimental::RNTupleInspector::Create(std::string_view ntupleName, std::s
 
 std::string ROOT::Experimental::RNTupleInspector::GetCompressionSettingsAsString() const
 {
-   int algorithm = fCompressionSettings / 100;
-   int level = fCompressionSettings - (algorithm * 100);
+   int algorithm = fCompressionSettings.value() / 100;
```

Comment:
Shouldn't this properly handle the case where compression is unknown, rather than throwing?

Reply:
Fixed.